TAO/tao/Strategies/advanced_resource.cpp

```diff
@@ -473,6 +476,10 @@ TAO_Advanced_Resource_Factory::allocate_reactor_impl () const
       break;
 #endif  /* ACE_HAS_EVENT_POLL || ACE_HAS_DEV_POLL */
 
+    case TAO_REACTOR_SINGLE_INPUT:
+      ACE_NEW_RETURN (impl, ACE_Single_Input_Reactor, 0);
```

Comment:
return nullptr instead of 0

Reply:
There are other `case`s in the existing code that use 0. I think this can be done in a separate PR to address all of them.

Reply:
Yes, there are other cases, just use nullptr in new code

Reply:
I have other changes to this coming soon, so I can change it.